sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcFileFormat.scala

```diff
@@ -83,6 +83,8 @@ class OrcFileFormat extends FileFormat with DataSourceRegister with Serializable
           classOf[MapRedOutputFormat[_, _]])
     }
 
+    org.apache.spark.sql.execution.datasources.orc.OrcFileFormat.checkFieldNames(dataSchema)
```

Comment:
Do we still need this when we move the check to `DataSourceAnalysis `?

Reply:
I see. I'll check this and remove it. Maybe, we can remove the similar logic from ParquetFileFormat, too.